src/main/java/com/epam/test_generator/dto/EditCaseDTO.java

```diff
@@ -1,18 +1,31 @@
 package com.epam.test_generator.dto;
 
+import com.epam.test_generator.entities.Action;
+import com.epam.test_generator.entities.Status;
+import java.util.Date;
 import javax.validation.constraints.Max;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.Size;
 
 public class EditCaseDTO {
 
+    private Long id;
+
     @Size(min = 1, max = 255)
     private String description;
 
     @Min(value = 1)
     @Max(value = 5)
     private Integer priority;
 
+    private Date creationDate;
+
+    private Date updateDate;
+
```

Comment:
Why do we need date fields in editDto class? Client should not set up this parameters.

Reply:
Agreed. No need to set it here

Reply:
Removed dates from EditCaseDTO